packages/ui/src/components/Question/Question.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { AccordionButton, AccordionIcon, AccordionItem, AccordionPanel } from '@chakra-ui/react';
+
+import { Typography } from '../Typography';
+
+export type QuestionProps = {
+  title: string;
+  content: string;
+};
+
+export const Question = ({ title, content }: QuestionProps) => {
+  return (
+    <AccordionItem borderColor="gray.200">
+      <h2>
+        <AccordionButton>
+          <Typography flex="1" textAlign="left" size="lg" weight="medium">
+            {title}
+          </Typography>
+          <AccordionIcon color="gray.400" />
```

Comment:
🤔 On mockups this icon is one of our icons instead of Chakra's default one and has 28x28px of size